docs/migration_ios.md

```diff
@@ -76,11 +76,11 @@ The chart below shows the new `Kin.start()` flow.
 
 ## Initiating the SDK
 The migration-enabled Dev Platform SDK adds a new property to the `Kin` class `var migrationDelegate: KinMigrationDelegate?`.
-This protocol allows you to hook into migration-related events (continue reading for more information).
+This protocol allows you to hook into migration-related events, and requires stubs to be implemented (continue reading for more information).
 
 ## The `KinMigrationDelegate` Protocol
 A protocol that allows you to hook into migration-related events.
-The protocol exposes the following methods:
+The protocol exposes the following new, REQUIRED methods, each of which *must be stubbed out* in the class you chose to implement the protocol in, regardless of whether you use them:
```

Comment:
1. This protocol is new to begin with, so saying that it exposes new methods is redundant. 
2. Also 'required' should be lowercase, but you can make it bold if you feel the need to emphasis it.
3. 'regardless of whether you use them' this is how Swift handles required protocols, is it really important to explain how the language works here?